vuepress/docs/next/docs/reference/bundle-upgrade.md

```diff
@@ -0,0 +1,141 @@
+---
+sidebarDepth: 2
+---
+
+# Bundle Upgrade Guideline
+
+This guideline describes the process for upgrading a git-based bundle to the newly introduced docker-based bundle (v5). 
+ 
+Git-based bundles do not need to be to upgraded. Both types of bundles are compatible with Entando 7.1+, but the v5 bundles provide several innovations and an improved developer experience.
+
+The improvements include:
+* A simplified bundle directory structure
+* A decoupled build and deployment process that is easier to understand, modify, and troubleshoot
+* CI/CD integration and support 
+* A single required Git source repository with Docker image repositories created automatically as needed
+* Optimized file storage and transfer methods using a layered file system, beneficial for large projects
+* A standardized bundle address format
+
+## Quick Reference
+* [Bundle Specifications](../curate/bundle-details.md)
+
+* [Bundle Management](../getting-started/ent-bundle.md)
+
+* [Bundle Version Comparison Table](./bundle-comparison.md)
+
+## Upgrade Process
+
+### 1. Initialize Bundle 
+
+Initialize a new bundle to set up the docker-based bundle structure using the ent CLI:
+
+``` 
+ent bundle init YOUR-BUNDLE-NAME
+```
+### 2. Copy Assets
+1. Use the ent CLI bundler command to export assets from an existing Entando Bundle. See the step-by-step [Export and Publish Tutorial](../../tutorials/create/pb/export-bundle-from-application.md). 
+
+2. The `platform` directory is designated for resources such as fragments, contents, pages, and templates. Copy the output of the ent bundler to 'YOUR-BUNDLE-NAME/platform`. 
+      
+>-You can also accomplish this manually, but the ent bundler can automatically generate top-level descriptors for platform resources.  
+-Widgets that are not MFEs are considered platform entities on Entando and should be placed in the `platform/widgets` directory.
+
+### 3. Add Micro Frontends & Microservices
+ 
+1. From inside your bundle project folder, add your micro frontend to the bundle descriptor `entando.json`:
+``` sh
+ent bundle mfe add YOUR-MFE1
+```
+>Docker-based bundles have a centralized bundle descriptor that includes specifications for bundle entities such as micro frontends, microservices, APIs, DBMS, and ID management. 
```

Comment:
"for bundle entities such as micro frontends, microservices, APIs, DBMS, and ID management" -> "for bundle entities such as micro frontends, microservices, and platform components."  Technically APIs, DBMS, and IDM are not first-class bundle entities.